src/inverseRobot.js

```diff
@@ -7,7 +7,17 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const newInverseRobot = {};
```

Comment:
```suggestion
  const inversedRobot = {};
```

Reply:
Avoid using `new` word in the variable naming, it's a kind of confusing

Reply:
fixed